agent/bench-scripts/pbench-linpack

```diff
@@ -158,7 +158,7 @@ function preprocess-iteration {
 }
 
 function postprocess-iteration {
-	local _cmd="${pbench_bin}/bench-scripts/postprocess/linpack-postprocess-cdm"
+	local _cmd="${pbench_bin}/bench-scripts/postprocess/linpack-postprocess"
```

Comment:
You additionally need to modify the `agent/config/benchmark/linpack.json` (also it's mentioned in the `agent/Makefile` and in a bunch of unittests.

Reply:
Good catch.

Reply:
Regarding `agent/config/benchmark/linpack.json`, ideally we would keep the interface the same for `pbench-run-benchmark`, but with it being deprecated, and having a working alternative here, I'll just remove the support for `linpack` under `pbench-run-benchmark`.